upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template

```diff
@@ -728,6 +777,10 @@ spec:
   {{ range $key, $value := . }}
     {{ $key }}: "{{ $value }}"
   {{ end }}
+{{ end }}
+{{ if and $spec.Kubelet $spec.Kubelet.ClusterDNS }}
```

Comment:
This is used by karpenter to configure kubelet on EKS through user data. It shouldn't apply to kOps.

Reply:
👍🏻 removed